tests/testthat/test-update_ard.R

```diff
@@ -0,0 +1,30 @@
+test_that("update_ard_fmt_fn()", {
+  expect_equal(
+    ard_continuous(ADSL, variables = AGE) |>
+      update_ard_fmt_fn(stat_names = c("mean", "sd"), fmt_fn = 8L) |>
+      apply_fmt_fn() |>
+      dplyr::filter(stat_name %in% c("mean", "sd")) |>
+      dplyr::pull("stat_fmt") |>
+      unlist(),
+    c("75.08661417", "8.24623390")
+  )
+
+  expect_snapshot(
+    error = TRUE,
+    ard_continuous(ADSL, variables = AGE) |>
+      update_ard_fmt_fn(stat_names = c("mean", "sd"), fmt_fn = -8L)
+  )
+})
+
+test_that("update_ard_stat_label()", {
+  expect_equal(
+    ard_continuous(ADSL, variables = AGE) |>
+      update_ard_stat_label(stat_names = c("mean", "sd"), stat_label = "Mean (SD)") |>
+      apply_fmt_fn() |>
```

Comment:
```suggestion
```

drop because it isn't needed here?

Reply:
I only added it to show the formatted result in the printed ARD. I think it's ok to keep it?